### PR TITLE
SFB-132/visual builder to v2

### PR DESCRIPTION
**Pull request by ZijneDwaasheid**
## ID
 SFB 132

 ## Description
Changing the builder tab so it supports and creates V2 forms

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

try making/editing V2 forms in the builder tab (no configuration or validations yet)

 ## Links to other PR's

### CODE REVIEW

File: public/forms/builder/form.ttl [hunk 1]
```diff
@@ -11,96 +11,96 @@
...
 ext:form a form:Form, form:TopLevelForm ;
-  form:includes ext:propertyGroupL;
+  form:includes ext:sectionL;
   form:includes ext:formNodesL;
   form:includes ext:formListingL;
   form:includes ext:formListingTableL.
 
 ##########################################################
-#  property-group
+#  properform:partOf
```
Comment (JonasVanHoof) [inline public/forms/builder/form.ttl:20]: Propertygroup is changed to section so make it Section

File: public/forms/builder/form.ttl [hunk 2]
```diff
@@ -229,7 +237,7 @@ ext:formListingFormItem a form:SubForm;
   form:includes ext:formListingAddF;
   form:includes ext:formListingRemoveF;
   form:includes ext:subformNodesL;
-  form:includes ext:subFormPropertyGroupF;
+  form:includes ext:subFormsectionF;
```
Comment (JonasVanHoof) [inline public/forms/builder/form.ttl:240]: Capital S

File: public/forms/builder/form.ttl [hunk 3]
```diff
@@ -250,17 +258,17 @@ ext:formListingRemoveF a form:Field ;
   sh:order 20 ;
   sh:path ( form:each form:removeLabel ) ;
   form:displayType displayTypes:defaultInput;
-  sh:group ext:listingPg .
+  form:partOf ext:listingPg .
 
-ext:subFormPropertyGroupF a form:Field ;
+ext:subFormsectionF a form:Field ;
```
Comment (JonasVanHoof) [inline public/forms/builder/form.ttl:263]: Capital S

**Review by JonasVanHoof — COMMENTED** [review]
Can create a V2 form with the builder. Just change the small things and it will be ok when looking at it now

File: app/components/rdf-form-fields/property-group-selector.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ export default class PropertyGroupSelector extends InputFieldComponent {
   loadOptions() {
     const sourceGraph = this.args.graphs.sourceGraph;
     const propertyGroup = new namedNode(
```
[the remark this answers is not in the document]
Comment (ZijneDwaasheid) [reply app/components/rdf-form-fields/property-group-selector.js:29]: changing variable names etc, I will do in another ticket later. It needs to be done in more places than only here. Just focusing on making correct V2 forms here

[the remark this answers is not in the document]
Comment (ZijneDwaasheid) [reply app/components/rdf-form-fields/property-group-selector.js:29]: https://binnenland.atlassian.net/browse/SFB-149?atlOrigin=eyJpIjoiZjgxMmYxYjk1YTU3NDhjMzkyOGUwYzhhNDFlNzdlOTUiLCJwIjoiaiJ9

**Review by JonasVanHoof — APPROVED** [review]

**Pull request merged**